src/tooltips/Tooltip.js

```diff
@@ -14,27 +14,9 @@ export default class Tooltip extends React.PureComponent {
   static propTypes = {
     hoverData: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
     layerProps: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
-    onMeasure: PropTypes.func.isRequired,
     theme: CustomPropTypes.theme.isRequired,
   }
```

Comment:
Can you compare this prop API against the one that existed for a custom tooltip before the refactor, and verify that it’s the same? This prop may not have been the only divergence

Reply:
Only other change was the `isRequired` on `theme`